### ISSUE TITLE
Get the Yeoman generator up to snuff

### ISSUE THREAD

**Issue by juliaelman** [18F/web-design-standards#1542]
The team has talked about creating a [Yeoman generator](http://yeoman.io/) to help users quickly stand up a website that uses the Standards. Here is an outline for the steps needed to accomplish this work.

- [x] Decide where the Yeoman generator should live
- [x] Create userflow to determine what should/needs to be included in the generator
- [ ] Implement (based off of userflow) 
- [ ] Test
- [ ] Iterate
- [ ] 💥

**Comment by Journerdism** [comment]
@juliaelman and @shawnbot Breaking this up a little more into sprint accomplishable tasks

**Comment by juliaelman** [comment]
@Journerdism @shawnbot and I broke up this larger issue into two smaller chunks for the Thanksgiving sprint milestone. We will be creating more issues to break down the implementation once we have a clear vantage point of what the users needs are for the Yeoman generator.

**Comment by juliaelman** [comment]
@Journerdism I also moved this into the Ninja Sprint, as Thanksgiving is upon us :turkey:

**Comment by shawnbot** [comment]
We are [cooking with 🔥 ](https://github.com/18F/generator-uswds)

**Comment by juliaelman** [comment]
After a team re-group/critique, we have decided not to move forward with the Yeoman generator method of creating an example project for the WDS. This is due to our current research around [Windows installation](https://github.com/18F/web-design-standards/issues/1533) and the difficulty we've seen users have in getting dependencies installed on their local machine. Adding the level of complexity of using Yeoman does not necessarily aid in users gaining a better understanding of getting up and running with the Standards, something which we hoped it would do.

Closing for now, but welcome folks to check out the [18F static site generator](https://github.com/18F/generator-18F-static) for implementing a Jekyll based static site with the Standards.
